implementations/rust/ockam/ockam_command/src/node/list.rs

```diff
@@ -141,18 +142,20 @@ impl Output for NodeListOutput {
             ),
         };
         let default = match self.is_default {
-            true => "(default)".to_string(),
+            true => " (default)".to_string(),
             false => "".to_string(),
         };
 
-        let output = format!(
-            r#"Node {node_name} {default} {status}
-{pid}"#,
-            node_name = self
-                .node_name
-                .to_string()
-                .color(OckamColor::PrimaryResource.color()),
-        );
+        let mut output = String::new();
+        write!(
```

Comment:
Was there some aspect that did not behave correctly when using `format`? If it works fine, I think we'd probably prefer to continue using that style and syntax here and only adjust the whitespace.

Reply:
In case you did this to avoid the weird new line formatting that we had before, you can use `formatdoc`:

```rust
let output = formatdoc! {"
    Node {node_name}{default} {status}
    {pid}",
    node_name = self
        .node_name
        .to_string()
        .color(OckamColor::PrimaryResource.color()),
};
```

Reply:
Thanks. My latest commit uses `formatdoc` instead to get around new line formatting. 

@shanesveller If you prefer I leave this implemented with `format` then also happy to do that. Let me know!

Reply:
No need if `formatdoc` also helped, but thanks for checking!